crates/ra_hir/src/ty/tests/data/0004_struct.txt

```diff
@@ -1,8 +1,8 @@
 [86; 90) 'C(1)': [unknown]
 [121; 122) 'B': B
 [86; 87) 'C': [unknown]
-[129; 130) '1': [unknown]
 [107; 108) 'a': A
+[129; 130) '1': [unknown]
```

Comment:
Perhaps it makes sense to sort by `(start_offset, length)`?

Reply:
Yeah, I didn't want to do it in the same commit since that would make it completely impossible to see the actual difference, but I should do it now...